themes/jaeger-docs/static/js/latest-articles.js

```diff
@@ -0,0 +1,56 @@
+function get_latest_post(){
+  fetch('https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/jaegertracing')
+   .then((res) => res.json())
+   .then((data) => {
+      const res = data.items
+      const posts = res.filter(item => item.categories.length > 0)
+      // Functions to create a short text out of whole blog's content
+      function toText(node) {
+         let tag = document.createElement('div')
+         tag.innerHTML = node
+         node = tag.innerText
+         return node
+      }
+      function shortenText(text,startingPoint ,maxLength) {
+         return text.length > maxLength?
+         text.slice(startingPoint, maxLength):
+         text
+      }
+
+      // Put things in right spots of markup
+      let output = '';
+      for(var i = 0; i<posts.length; i++){
+        console.log(posts[i].link)
+         output += `
+         <li class="blog__post">
+            <a href="${posts[i].link}">
+
+               <img src="${posts[i].thumbnail}" class="blog__topImg"></img>
+
+
+               <div class="blog__content">
+                  <div class="blog_preview">
+                     <h2 class="blog__title">${shortenText(posts[i].title, 0, 30)+ '...'}</h2>
+
+                     <p class="blog__intro">${'...' + shortenText(toText(posts[i].content),60, 300)+ '...'}</p>
+                  </div>
+                  <hr>
+                  <div class="blog__info">
+                     <span class="blog__author">${posts[i].author}</span>
+                     <span class="blog__date">${shortenText(posts[i].pubDate,0 ,10)}</span>
+                  </div>
+
+               </div>
+
+
+            <a/>
+         </li>`
+
+        }
+        document.querySelector('.blog__slider').innerHTML = output
+        $(document).ready(function () {
+            $("#blog").show();
+        });
+  })
+}
+get_latest_post()
```

Comment:
Is there any way this function can blow up?  If it can, I would prefer to have a try/catch and log the error (or report to google analytics, not sure if we even have it enabled), but not break the rest of the home page.

Reply:
With the approach that @lucperkins outlined, it would still work even if Medium is down. Would you be willing to give it a shot?

Reply:
Netlify will update the website whenever a new commit arrives at this repository. IMO, it should be sufficient (for now) to have the script to open a PR and wait for someone to approve/merge. Once we get confident enough, we can think of something to get the PR merged automatically.

Reply:
Yes, this is a different "script", but probably very similar to the latest newsletter.

Reply:
I would still like to merge this first, not block on doing the webhook approach.